### DISCUSSION TITLE
[Bug] Incorrect display of wallet balance.

### DISCUSSION THREAD

**Discussion by Johnybcn777** [Chia-Network/chia-blockchain#11425] [General]
### What happened?

Incorrect display of wallet balance. After the blockchain was restarted, the funds credited from 04/24/2022 to 04/28/2022 in the total amount of 0.15 chia disappeared from the wallet. Could you explain why this happened and when the missing funds will be returned to my wallet???

Visualización incorrecta del saldo de la billetera. Después de reiniciar la cadena de bloques, los fondos acreditados del 24/04/2022 al 28/04/2022 por un monto total de 0,15 chia desaparecieron de la billetera. ¿Podría explicar por qué sucedió esto y cuándo se devolverán los fondos perdidos a mi billetera?

### Version

1.3.4

### What platform are you using?

Windows

### What ui mode are you using?

GUI

**Comment by davision** [comment]
The wallet needs to resync. After that, your funds will show up.

**Comment by Johnybcn777** [comment]
What do I have to do to resynchronize?

**Comment by shaneo257** [comment]
close the client, delete the wallet db, and let it sync from 0

**Comment by kckc33** [comment]
i am having the same issue
1-i delete everything of chia and reinstalled
still getting wrong balance and it kept saying my wallet is already syned
2-i log out and delete wallet again log back in , it appears i am syned again and balance is still wrong

and there is no transaction history as well

what shoudl i do/

**Comment by Johnybcn777** [comment]
I turned off the blockchain and erased the database, then restarted everything and the synchronization started. After 5 minutes, the wallet and the balance were restored as it was before the loss. The entire blockchain is still being synchronized. I didn’t download the database, because there may be an error in the file and before that I just downloaded the database so as not to wait for full synchronization and after 2 weeks the coins flew off for this reason now I’ll wait for full synchronization

**Comment by kckc33** [comment]
i restarted everything,,,
and 3 seconds the wallets show it is syned 
but shows nothing 
oh man.........what should i do....

**Comment by emlowe** [comment]
Possibly fixed in https://github.com/Chia-Network/chia-blockchain/pull/11338

**Comment by yalin161815** [comment]
我的钱包1.26 升级到1.32 后，接收地址发生改变，余额转到之前接收的地址上面，现在钱包的余额为零，怎么把之前的币转出来
